Increase worker timout

While running this on Rpi4 (2GB RAM, 4GB SWAP on SSD), I see all of workers getting timeout and eventually no machine learning services working. 
```
[2023-09-22 23:48:13 +0530] [2767] [INFO] Booting worker with pid: 2767
[2023-09-22 23:48:43 +0530] [1305] [CRITICAL] WORKER TIMEOUT (pid:2767)
[2023-09-22 23:48:45 +0530] [1305] [ERROR] Worker (pid:2767) was sent SIGKILL! Perhaps out of memory?
[2023-09-22 23:48:45 +0530] [2821] [INFO] Booting worker with pid: 2821
[2023-09-22 23:49:15 +0530] [1305] [CRITICAL] WORKER TIMEOUT (pid:2821)
[2023-09-22 23:49:16 +0530] [1305] [ERROR] Worker (pid:2821) was sent SIGKILL! Perhaps out of memory?
[2023-09-22 23:49:16 +0530] [2873] [INFO] Booting worker with pid: 2873
[2023-09-22 23:49:46 +0530] [1305] [CRITICAL] WORKER TIMEOUT (pid:2873)
```
looking further, I see similar issue was fixed in docker compose way 
[fix(ml): set higher default worker timeout](https://github.com/immich-app/immich/pull/4007)

Do you know how to pass these parameter or just increase timeout in single docker way ?

will it be here? [root/etc/s6-overlay/s6-rc.d/svc-machine-learning/run](https://github.com/imagegenius/docker-immich/blob/a74e3f93df597e60e4fe878f3cab6bb78a3667cf/root/etc/s6-overlay/s6-rc.d/svc-machine-learning/run#L17)

--- a/readme-vars.yml
+++ b/readme-vars.yml
@@ -47,6 +47,8 @@ opt_param_env_vars:
   - { env_var: "DB_PORT", env_value: "5432", desc: "PostgreSQL Port" }
   - { env_var: "REDIS_PORT", env_value: "6379", desc: "Redis Port" }
   - { env_var: "REDIS_PASSWORD", env_value: "", desc: "Redis password" }
+  - { env_var: "MACHINE_LEARNING_WORKERS", env_value: "1", desc: "Machine learning workers" }
+  - { env_var: "MACHINE_LEARNING_WORKER_TIMEOUT", env_value: "120", desc: "Machine learning worker timeout" }
 opt_param_usage_include_vols: true
 opt_param_volumes:
   - { vol_path: "/config/machine-learning", vol_host_path: "path_to_machine-learning", desc: "Store the machine-learning models (~800MB)" }
@@ -124,6 +126,7 @@ external_application_unraid_block: |
 
 # changelog
 changelogs:
+  - { date: "24.09.23:", desc: "add vars for ml workers/timeout" }
   - { date: "29.07.23:", desc: "remove cuda acceleration for machine-learning" }
   - { date: "23.05.23:", desc: "move to ubuntu lunar and support cuda acceleration for machine-learning" }
   - { date: "22.05.23:", desc: "deprecate postgresql docker mod" }

--- a/root/etc/s6-overlay/s6-rc.d/svc-machine-learning/run
+++ b/root/etc/s6-overlay/s6-rc.d/svc-machine-learning/run
@@ -18,4 +18,5 @@ exec \
         -k uvicorn.workers.UvicornWorker \
         -w 1 \
         -b 0.0.0.0:3003 \
+        -t 120 \
         --log-config-json log_conf.json

--- a/root/etc/s6-overlay/s6-rc.d/svc-machine-learning/run
+++ b/root/etc/s6-overlay/s6-rc.d/svc-machine-learning/run
@@ -16,7 +16,7 @@ exec \
     cd /app/immich/machine-learning s6-setuidgid abc \
        gunicorn app.main:app \
         -k uvicorn.workers.UvicornWorker \
-        -w 1 \
+        -w ${MACHINE_LEARNING_WORKERS:=1} \
         -b 0.0.0.0:3003 \
-        -t 120 \
+        -t ${MACHINE_LEARNING_WORKER_TIMEOUT:=120} \
         --log-config-json log_conf.json

